src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -48,6 +48,27 @@ To leave the cluster untouched by the execution finish you should export `TEST_T
 
 To use existing cluster which have already installed Cloud API Adaptor, you should export `TEST_INSTALL_CAA=no`.
 
+## Attestation and KBS specific
+
+To deploy the KBS service and test attestation related cases, set `DEPLOY_KBS=yes`.
+To use a customized OPA policy file in KBS service, specify the OPA file via `TEE_CUSTOMIZED_OPA=/path/policy.rego`.
+For example, we need set `TEE_CUSTOMIZED_OPA=/$trustee_path/kbs/sample_policies/allow_all.rego` when test sample TEE as default policy disabled sample TEE.
```

Comment:
Can we move this section below the artifacts code so that we have checked out the trustee code first?

Reply:
Fixed, thanks! Steve.